### ISSUE TITLE
What about streaming?

### ISSUE THREAD

**Issue by cabrinoob** [dmanjunath/node-redshift#3]
Is there a way to stream out rows from redshift instead of getting the entire dataset into memory before return it to client?

Thank you for your help

**Comment by dmanjunath** [comment]
Yeah! Theoretically there's a way to do this using redshift cursors and node streams like in this module https://github.com/brianc/node-pg-query-stream. A streaming API is a great feature idea but this project currently doesn't have support for it. But I'm going to put it on the feature list for the next version.

**Comment by raffacabofrio** [comment]
This would be nice.

I need to write a very large csv. And would be nice to append row by row.
